lib/dataElementListener.js

```diff
@@ -371,7 +371,19 @@ class DataElementImporter extends SHRDataElementParserListener {
           const onValue = cst.elementTypeConstraint().KW_ONLY() ? true : false;
           value.addConstraint(new TypeConstraint(newIdentifier, path, onValue));
         }
-      } else if (cst.elementIncludesTypeConstraint()) {
+      } else if (cst.elementSubsetConstraint()) {
+        let subsetList = [];
+        for (const subsetConstraint of cst.elementSubsetConstraint().subsetConstraint()) {
+          const newIdentifier = this.resolveToIdentifierOrTBD(subsetConstraint);
+          if (newIdentifier.isSpecialKeyWord) {
+            logger.error(`Fields cannot be constrained to type "${newIdentifier.name}". ERROR_CODE:11025`);
```

Comment:
Ha.  My turn to catch _you_ using the old logger format. ;-)

Reply:
I played myself! Fixed it here https://github.com/standardhealth/shr-text-import/pull/108/commits/a4cbf07a57cfcc7dcb0f568e31f5b545da11edd1.